src/main/resources/application.yml

```
@@ -1,6 +1,6 @@
 spring:
   profiles:
-    active: ${SPRING_PROFILES_ACTIVE:dev}  #
+    active: ${SPRING_PROFILES_ACTIVE:prod}  #
```

```suggestion
     active: ${SPRING_PROFILES_ACTIVE:dev}
```
기본 yml은 prod로 하시기보다 dev로 설정하시는것이 좋습니다.
지금은 모든 설정이 prod로 되어있는것 같은데, 로컬에서 돌리는 경우에도 prod가 붙으므로 위험한 설정입니다.

---

넵 알겠습니다. 수정하겠습니다